tests/integration/components/frost-text-test.js

```diff
@@ -16,29 +19,301 @@ describeComponent(
     })
 
     it('renders', function () {
-      this.render(hbs`{{frost-text}}`)
-      expect(this.$()).to.have.length(1)
+      this.render(hbs`
+          {{frost-text}}
+      `)
+
+      expect(
+        this.$('input').attr('tabindex'),
```

Comment:
Lets change this to use the `.prop()` method instead of `.attr()` then once the prop-types is set to number and the default is set to `0` you can change the expect to `.to.eql(0)`